faros_event.sh

```diff
@@ -168,7 +170,7 @@ main() {
             log "$http_response_body"
         fi
     else
-        log "Dry run: Event NOT sent to Faros."
+        log "Dry run and/or Community edition: Event NOT sent to Faros."
```

Comment:
Could we be consistent with dry_run functionality across both our standard events and the CE mutations? Meaning something like this:
```sh
if !(($community_edition)); then
  log "Request Body:"
  log "$request_body"

  if !(($dry_run)); then
  ... standard event stuff
  else
  log "Dry run: Event NOT sent to Faros."
  fi
else
  doCIMutations # I think this might make more sense here
fi
```
This way we don't log a weird empty request body. I do however want to log the request body of the mutations even when it is a `dry_run` see related comment below.

Reply:
That's much better, thanks!